@utahdts/utah-design-system/@types/jsDocTypes.d.js

```diff
@@ -406,6 +411,7 @@
  *  @property {string} title
  *  @property {string[]} [parentLinks]
  *  @property {WebsiteMainMenuItem[]} [children]
+ *  @property {'inline' | 'flyout' | 'mega-menu' | 'plain' | null} [childrenMenuType]
```

Comment:
type could be {ChildrenMenuTypes | null}

probably would need to `/** @typedef {import ('@utahdts/utah-design-system-header').ChildrenMenuTypes}ChildrenMenuTypes */` at the top of the file?